index.bs

```diff
@@ -1860,10 +1871,10 @@ To <dfn export>process an attribution eligible response</dfn> given a [=suitable
     "<code>[=eligibility/event-source-or-trigger=]</code>".
 1. Let |reportingOrigin| be |response|'s [=response/URL=]'s [=url/origin=].
 1. If |reportingOrigin| is not [=check if an origin is suitable|suitable=], return.
-1. Let |sourceHeader| be the result of [=header list/get|getting=]
+1. Let |sourceHeader| be the result of [=get non-coalesced header|getting=]
     "`Attribution-Reporting-Register-Source`" from |response|'s
     [=response/header list=].
-1. Let |triggerHeader| be the result of [=header list/get|getting=]
+1. Let |triggerHeader| be the result of [=get non-coalesced header|getting=]
```

Comment:
I don't think this properly handles:

```http
Attribution-Reporting-Register-Source: {...}
Attribution-Reporting-Register-Source: {...}
Attribution-Reporting-Register-Trigger: {...}
```
In this case, the two sources will be ignored but the trigger will be handled. IMO this should be an error that causes all the headers to be ignored instead.